CONTRIBUTING.md

```diff
@@ -144,6 +144,7 @@ Because SQL within JSON or YAML will not generally be syntax highlighted, indent
 * Icons must use SVG fills only, not SVG strokes or any feature Mapnik does not support.
 * Use no color for the icon's fill if the icon is monochromatic. This allows the color to be set in the MSS.
 * Use a common canvas size, which is usually 14x14 px.
+* Convert shapes and other components to paths and merge them into a compound path.
```

Comment:
We don't need compound paths unless there are holes in a desired shape. Also, I thought shapes like rect and circle worked fine.